app/views/courses/PodcastItemContainer.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <div v-if="podcastsLoaded && latestPodcast">
+      <podcast-item-component :podcast="latestPodcast"/>
+    </div>
+  </div>
+</template>
+
+<script>
+import store from 'core/store'
+import cocoVueRouter from 'app/core/vueRouter'
+import { mapActions, mapGetters } from 'vuex'
+import PodcastItemComponent from 'app/views/podcast/PodcastItemComponent'
+import podcastVisibleMixin from 'app/views/podcast/podcastVisibleMixin'
+
+export default Vue.extend({
+  name: 'PodcastItemContainer',
+  store,
+  components: {
+    'podcast-item-component': PodcastItemComponent
+  },
+  data () {
+    return {
+      podcastsLoaded: false
+    }
+  },
+  router: cocoVueRouter(),
```

Comment:
Because the `<router-link` in `PodcastItemComponent.vue` will only work if we have this. 
Normally it's included in `app/views/core/SingletonAppVueComponentView.js`, but we need this here because this component is injected direclty into a backbone comopnent's pug template (without vueRouter)